activejob/test/helper.rb

```diff
@@ -12,6 +12,7 @@
   require "support/integration/helper"
 else
   ActiveJob::Base.logger = Logger.new(nil)
+  ActiveJob::Base.skip_after_callbacks_if_terminated = true
```

Comment:
We seem to have a bunch of job in the test suite that trigger deprecation, I can fix them one by one if you prefer but I figured it was easier to just set the flag to true (especially since the tests doesn't rely on whether after_x callbacks are executed)